### PR TITLE
Add real life example to comms package readme

### LINKED ISSUE #64: Add an real life example for blocs to comms package README
Add to readme example showing communication between `AuthCubit` and `UserProfileCubit` implemented with comms pattern.

### PR DESCRIPTION
**Pull request by lewandowski-jan**
Closes #64

### CODE REVIEW

File: packages/comms/README.md [hunk 1]
```diff
@@ -290,6 +290,89 @@ void main() async {
 }
 ```
 
+### Real life example
+
+Most common bloc that each app has is `AuthBloc` or `AuthCubit`. It's important
+for all other blocs to know and react to whether user is authenticated or not.
+All you need to do is add a `StateSender` mixin which allows every other bloc
+to listen for `AuthState` changes.
```
Comment (bartekpacia) [inline packages/comms/README.md:298]:
```suggestion
With `comms`, all you need to do is add a `StateSender` mixin to the auth bloc, which allows every other bloc to listen for `AuthState` changes.
```

File: packages/comms/README.md [hunk 2]
```diff
@@ -290,6 +290,89 @@ void main() async {
...
+
+```dart
+class AuthCubit extends Cubit<AuthState> with StateSender {...}
+```
+
+For example `UserProfileCubit` will fetch user's profile on sign in and
+should clear that data on sign out.
```
Comment (bartekpacia) [inline packages/comms/README.md:305]:
```suggestion
For example `UserProfileCubit` will fetches user's profile on sign in and
clears that data on sign out.
```

**Review by bartekpacia — DISMISSED** [review]
👍🏻

File: packages/comms/README.md (hunk 2 above)
Comment (PiotrRogulski) [reply packages/comms/README.md:305]:
```suggestion
For example, `UserProfileCubit` fetches the user's profile on sign-in and
clears that data on sign-out.
```

File: packages/comms/README.md [hunk 3]
```diff
@@ -290,6 +290,89 @@ void main() async {
...
+       super(const UserProfileState.initial());
+  
+  final UserRepository _repository;
+
+  @override
+  void onInitialMessage(AuthState message) => onMessage(message);
+
+  @override
+  void onMessage(AuthState message) {
+    message.maybeWhen<void>(
```
Comment (PiotrRogulski) [inline packages/comms/README.md:321]: `.maybeWhen` is specifically generated by `freezed` – consider using a `switch` or an `if` instead

**Review by PiotrRogulski — DISMISSED** [review]

**Review by PiotrRogulski — APPROVED** [review]

**Pull request merged**